arbitrum-docs/build-decentralized-apps/token-bridging/layer-leap.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: 'Layer leap contracts'
+description: 'Learn to use Layer leap contracts.'
+author: leerderek
+sme: amarrazza
+target_audience: 'Developers who want to implement Layer leap contracts on their Orbit chain'
+sidebar_position: 1
+---
+
+import PublicPreviewBannerPartial from '../../partials/_public-preview-banner-partial.mdx';
+
+<PublicPreviewBannerPartial />
+
+## Single click bridging from L1 to L3 is now possible with layer leap
+
+- [ Main repository ](https://github.com/OffchainLabs/l1-l3-teleport-contracts) 
+- [ SDK support ](https://www.npmjs.com/package/@arbitrum/sdk/v/3.5.1-teleporter.0) (pre-release!)
+- [ Request Feature ](https://github.com/OffchainLabs/arbitrum-token-bridge/issues/new?assignees=&labels=feat,triage&projects=&template=add-layerleap-request.yml&title=[feat]) 
+
+Contract Addresses
+
+| Contract              | Address                                                                 |
+| --------------------- | ----------------------------------------------------------------------- |
+| Arb One L1Teleporter  | https://etherscan.io/address/0xCBd9c6e310D6AaDeF9F025f716284162F0158992 |
+| Arb Nova L1Teleporter | https://etherscan.io/address/0xCBd9c6e310D6AaDeF9F025f716284162F0158992 |
+
+## Launch details
+
+_Key timelines & announcements_
+
+**Orbit Onboarding:** We are looking for interested Orbit L3s with ETH gas for immediate rollout **AND** Orbit L3s with custom gas tokens for future adoption in our next release.
+
+_Upcoming launches & future releases_
+
+A future release of Layer Leap is not yet scheduled, but upcoming features including providing support for Orbit L3s with custom gas tokens.
+
+## Value & user impact
+
+**End-users:** Can now bridge `ETH` and `ERC-20`s from Ethereum to your Orbit L3 in a single click.
+
+- Bridging `ETH` requires one call on L1
+- Bridging an `ERC-20` requires one approval and one call on L1
+
+**Orbit L3s:** Can more seamlessly onboard `ETH` and `ERC-20`s from Ethereum mainnet.
+
+_Which Orbit configurations can adopt this feature?_
+
+**Any L3 Orbit Chains on top of an Arbitrum chain** **_with ETH gas fees_** can use Layer Leap.
```

Comment:
might be worth stating that L3s with ETH gas can use it today and L3s with custom gas tokens is on our roadmap.